interactive/templates/interactive/new_analysis_request.html

```diff
@@ -24,50 +25,72 @@ <h1 class="text-center text-3xl font-extrabold text-slate-900 sm:w-full sm:max-w
   {{ form.non_field_errors }}
 
   <div>
     {% include "components/form-field.html" with id=form.title.id_for_label label=form.title.label name="title" required=True type="text" hint="For example: Systolic code usage" %}
+    {% if form.title.errors %}
+      <ul class="mt-2">
+        {% for error in form.title.errors %}
+          <li class="text-sm text-red-700">{{ error }}</li>
+        {% endfor %}
+      </ul>
+    {% endif %}
   </div>
 
-  <div x-title="codelist-select" x-data="{
-      codelist: '',
-      get selectedCodelist() {
-        var select = document.getElementById('{{ form.codelist_slug.id_for_label }}');
-        return {
-          url: `https://www.opencodelists.org/codelist/${this.codelist}`,
-          text: `View &#8220${select.options[select.selectedIndex].text}&#8221 codelist`
-        }
-      }
-    }">
-    {% include "components/form-field-select.html" with id=form.codelist_slug.id_for_label label="Codelist" name="codelist_slug" required=True choices=form.codelist_slug.field.choices hint="Codelists are published by OpenSAFELY on OpenCodelists and use the SNOMED CT coding system" x_model="codelist" %}
-
-    <div class="border-l-2 border-l-oxford-200 bg-oxford-50 rounded pl-4 pr-2 py-4 mt-4 text-sm text-slate-800">
-      <ul class="list-disc pl-4 grid grid-flow-row gap-2">
-        <li>
-          {% include "components/link.html" with href="https://www.opencodelists.org/docs/#what-is-a-codelist" content="What is a codelist?" newtab=True %}
-        </li>
-        <li>
-          {% include "components/link.html" with href="https://www.snomed.org/" content="What is SNOMED CT?" newtab=True %}
-        </li>
-        <li x-show="codelist">
-          <a
-            class="
-              text-oxford-600 font-semibold underline underline-offset-1 transition-colors
-              hover:text-oxford-700 hover:no-underline
-              focus:text-oxford-900 focus:no-underline
-            "
-            rel="noopener noreferrer"
-            target="_blank"
-            x-bind:href="selectedCodelist.url"
-            x-text="selectedCodelist.text"
-            ></a>
-        </li>
-      </ul>
+  {{ form.codelist_slug.field.choices|json_script:"react-dropdown-options" }}
+  {{ form.codelist_slug.errors|json_script:"react-dropdown-errors" }}
+  <div
+    data-choices="react-dropdown-options"
+    data-errors="react-dropdown-errors"
+    data-hint="Codelists are published by OpenSAFELY on OpenCodelists and use the SNOMED CT coding system"
+    data-id="{{ form.codelist_slug.id_for_label }}"
+    data-label="Codelists"
+    data-name="codelist_slug"
+    id="react-dropdown"
+    >
+      {% include "components/form-field-select.html" with id=form.codelist_slug.id_for_label label="Codelist" name="codelist_slug" required=True choices=form.codelist_slug.field.choices hint="Codelists are published by OpenSAFELY on OpenCodelists and use the SNOMED CT coding system" %}
     </div>
-  </div>
+  {% if debug %}
```

Comment:
Is this debug section something that's going to be needed by other pages and if so, would it make sense to move into base.html?